### PR TITLE
[ve-aphra] add veAPHRA

### PR DESCRIPTION
**Pull request by androolloyd**
veAPHRA is the vote escrow nft of AphraFinance that holds the underlying vote weights, this subgraph computes the total voting power for all owned nfts by any target addresses at any target block including latest

### CODE REVIEW

**Comment by ChaituVR** [comment]
Hi @androolloyd Make sure you add strategy to src/strategies/index.ts and make all chars in strategy name are lowercased

**Comment by androolloyd** [comment]
@ChaituVR should be good to go

File: src/strategies/ve-aphra/examples.json [hunk 1]
```diff
@@ -0,0 +1,542 @@
...
+    },
+    "network": "1",
+    "addresses": [
+      "0x86d3ee9ff0983bc33b93cc8983371a500f873446",
+      "0xded5d8c6b271a48f3b9ba20e8aa761a6baa2fac9",
+      "0x7f82e84c2021a311131e894ceff475047ded4673",
+      "0xd6bf9d9079139dbb3a4fdb84b07efc75ea77838f",
+      "0x8b4372ce449f673761bf9f98f5d27ad1b3bd02c8",
+      "0x83dedbb4d75df6ea216ccf3af93a7995f3ae46d1",
+      "0x9d63a78f6195dd1d5fc42bcd32b5014dea6b71ae",
```
[the remark this answers is not in the document]
Comment (androolloyd) [reply src/strategies/ve-aphra/examples.json:20]: Ok I'll update I set to over 500 to hit all the strategy tests.

[the remark this answers is not in the document]
Comment (ChaituVR) [reply src/strategies/ve-aphra/examples.json:20]: @androolloyd to test 500 addresses, you can run `yarn test --strategy=ve-aphra --more=500`

**Comment by ChaituVR** [comment]
Closing this as it is stale, feel free to reopen if needed

**Pull request closed without merging**